src/TcUnit.TestAdapter/Execution/TestRunner.cs

```diff
@@ -158,15 +158,7 @@ public TestRun RunTests(TwinCATXAEProject project, IEnumerable<TestCase> tests,
             var target = AmsNetId.Parse(runSettings.Target);
             var cleanUpAfterTestRun = runSettings.CleanUpAfterTestRun;
             var cleanUpBeforeTestRun = true;
-
-            if(AmsNetId.LocalHost != target && AmsNetId.Local != target)
```

Comment:
Here I am removing this check since the usermode runtime doesn't appeart in the list of remote routes, and it does not have a local host net ID.

@philippleidig is this OK or do you have a suggestion on how to handle this?

Reply:
Totally fine for me. To be honest, I don't have any other solution either.